### PR TITLE
Change seen user list to React component

### PR DESCRIPTION
**Pull request by okonomi**
- Change seen user list to React component.
- New features are not implemented. Just refactoring only.
  - However, I added a size property to UserPicture component(need xs size picture).

### CODE REVIEW

File: resource/js/components/UserList.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    const users = this.props.users.map((user) => {
+      return (
+        <a data-user-id={user._id} href={'/user/' + user.username} title={user.name}>
+          <UserPicture user={user} size="xs" />
+        </a>
+      );
+    });
+
+    return (
+      <p className="seen-user-list">
```
Comment (sotarok) [inline resource/js/components/UserList.js]: If this `UserList` component is just for `seen-user-list`, the component should be located under `component/SeenUserList/` dir.
As you know, only the root components are allowed to locate direct under `component/` dir.

File: resource/js/components/SeenUserList.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        // ignore unless response has error
+        if (res.ok) {
+          this.setState({seenUsers: res.users});
+        }
+      }.bind(this));
+    }
+  }
+
+  getSeenUserIds() {
+    const $seenUserList = $("#seen-user-list");
```
Comment (sotarok) [inline resource/js/components/SeenUserList.js]: umm ... this `$` is ok if it is the temporary implementation.
Should leave `// FIXME` comment here ... 💃 or I would like to think of another way to bind value.

File: resource/js/components/SeenUserList.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import UserList from './UserList';
+import $ from 'jquery';
```
Comment (okonomi) [inline resource/js/components/SeenUserList.js]: Removed.

File: resource/js/components/SeenUserList.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    this.state = {
+      seenUsers: [],
+    };
+  }
+
+  componentDidMount() {
+    const seenUserIds = this.getSeenUserIds();
+    if (seenUserIds.length > 0) {
+      // FIXME: user data cache
+      $.get('/_api/users.list', {user_ids: seenUserIds.join(',')}, function(res) {
```
Comment (okonomi) [inline resource/js/components/SeenUserList.js]: It replaced `Crowi.apiGet()`.

File: resource/js/components/User/UserPicture.js [hunk 5]
```diff
@@ -13,23 +13,34 @@ export default class UserPicture extends React.Component {
...
+
   render() {
     const user = this.props.user;
 
     return (
       <img
         src={this.getUserPicture(user)}
         alt={user.username}
-        className="picture picture-rounded"
+        className={this.getClassName()}
```
Comment (okonomi) [inline resource/js/components/User/UserPicture.js]: 😄

File: resource/js/components/SeenUserList.js (hunk 2 above)
Comment (okonomi) [inline resource/js/components/SeenUserList.js]: Add FIXME comment.

File: resource/js/components/UserList.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import UserPicture from './User/UserPicture';
+
+export default class UserList extends React.Component {
+
+  render() {
+    const users = this.props.users.map((user) => {
+      return (
+        <a data-user-id={user._id} href={'/user/' + user.username} title={user.name}>
```
Comment (okonomi) [inline resource/js/components/UserList.js]: Add `key` attr.

File: resource/js/components/UserList.js (hunk 1 above)
Comment (okonomi) [inline resource/js/components/UserList.js]: I think `UserList` can be used not only for `SeenUserList` but also for "Like" user list.
Where should such commonly usable components be placed?

Comment (okonomi) [inline resource/js/components/UserList.js]: OK.
Move `UserList` to `SeenUserList/` directory.

**Pull request merged**